dashboard/app/models/workshops_cohort.rb

```diff
@@ -0,0 +1,5 @@
+class WorkshopsCohort < ActiveRecord::Base
```

Comment:
I would recommend renaming this class to WorkshopCohort, then you don't need the self.table_name line below.
(would also have to rename workshops_cohorts -> workshop_cohorts above)